lib/ice_cube/rules/daily_rule.rb

```diff
@@ -11,6 +20,26 @@ def initialize(interval = 1)
       reset
     end
 
+    def verify_alignment(value, freq, rule_part)
+      return unless freq == :wday || freq == :day
+      return unless @validations[:interval]
+
+      interval_validation = @validations[:interval].first
```

Comment:
Maybe worth moving this interval-checking into a util that we can use in both places

Reply:
Yes, I'm not 100% happy with some of the duplication & super-calling either. I'll see what I can do.